test/json-ld.net.tests/ConformanceTests.cs

```diff
@@ -80,13 +77,18 @@ public IEnumerator<object[]> GetEnumerator()
             foreach (string manifest in manifests)
             {
                 JToken manifestJson = jsonFetcher.GetJson(manifest, rootDirectory);
+                var isRemoteTest = (string)manifestJson["name"] == "Remote document";
 
                 foreach (JObject testcase in manifestJson["sequence"])
                 {
                     Func<JToken> run;
                     ConformanceCase newCase = new ConformanceCase();
 
-                    newCase.input = jsonFetcher.GetJson(testcase["input"], rootDirectory);
+                    // Load input file if not remote test. Remote tests load from the web at test execution time.
+                    if (!isRemoteTest)
+                    {
+                        newCase.input = jsonFetcher.GetJson(testcase["input"], rootDirectory);
+                    }
```

Comment:
Turns out this was depending on the overly broad `catch` in `JsonFetcher.cs`. I made this more precise, and changed `JsonFetcher` to return NULL iff the JSON failed to parse, which seems like reasonable for a test loader method.

Reply:
i am still not sure that we should have a loader in the test project (not that it's the concern of this PR). Shouldn't this be part of what we are testing?

Reply:
I looked at JSONUtils to see if any of its `From*()` methods could help. There isn't one that opens a file, which is: a) what we would need in order to replace this, and b) essentially what `JsonFetcher` appears to do.